### PR TITLE
feat(dedicated.nutanix): changes related to nutanix scale up and down

### PR DESCRIPTION
**Pull request by sachinrameshn**
| Question         | Answer
| ---------------- | ---
| Branch?          | `release/nutanix-scale-up-and-down`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-10824
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [ ] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~ [n/a]

## Description

Changes on manager nutanix side due to Nutanix scale up and down.

## Related

- [x] TRANS-52536

### CODE REVIEW

File: packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/constants.js [hunk 1]
```diff
@@ -3,9 +3,10 @@ export const TRAVAUX_LINK = 'https://www.status-ovhcloud.com/';
 export const NUTANIX_NODE_STATUS = {
   DONE: 'DONE',
 };
-
+export const PRIVATE_BANDWIDTH_SERVICE_NODE_PREFIX = 'vrack-bandwidth';
```
Comment (anooparveti) [inline packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/constants.js:6]: **question:** Dont we need to include `cluster-vrack-bandwidth`?

File: packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/controller.js [hunk 2]
```diff
@@ -2,7 +2,15 @@ import { NO_OS_INSTALLED_REGEX, TRAVAUX_LINK } from './constants';
 
 export default class NutanixNodeGeneralInfoCtrl {
   /* @ngInject */
-  constructor($translate, coreConfig, NutanixNode, $rootScope, Alerter) {
+  constructor(
+    $state,
```
Comment (anooparveti) [inline packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/controller.js:6]: **nitpick:** Dont inject `$state` into controller. Can you please get the `href` from resolve?

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/constants.js (hunk 1 above)
Comment (sachinrameshn) [reply packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/constants.js:6]: Hello Anoop, For new catalog bandwidth plan codes have been renamed from "cluster-vrack-bandwidth" to "vrack-bandwidth".

File: packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/controller.js (hunk 2 above)
Comment (sachinrameshn) [reply packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/controller.js:6]: sure

**Review by qpavy — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/routing.js [hunk 3]
```diff
@@ -58,7 +65,19 @@ export default /* @ngInject */ ($stateProvider) => {
...
         NutanixService.getBandwidthOptions(nodeId),
-
+      serviceInfos: /* @ngInject */ ($stateParams, NutanixNode) =>
+        NutanixNode.getServiceInfos($stateParams.nodeId).then(
+          (serviceInfo) => ({
+            ...serviceInfo,
+            serviceType: 'DEDICATED_SERVER',
+          }),
+        ),
+      atTrack: /* @ngInject */ (atInternet) => (name) =>
```
[the remark this answers is not in the document]
Comment (sachinrameshn) [reply packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/routing.js]: Done Anoop

**Pull request merged**